go/cmd/deploy/main.go

```
@@ -119,55 +119,36 @@ var DeployFlags = []cli.Flag{
 	cli.String("auth-token", "Control plane auth token", cli.Default(DefaultAuthToken)),
 }
 
+// WARNING: Changing the "Description" part will also affect generated MDX.
 // Cmd defines the deploy CLI command
 var Cmd = &cli.Command{
```

golangci-lint (2.2.2)

cli.Command is missing fields Version, Commands, Aliases

(exhaustruct)